### PR TITLE
Feature/set weights and bias

### PR DESCRIPTION
**Pull request by Wissiak**
First of all, thank you for this awesome neural network playground 🤩

For a school exercise I performed backpropagation calculations and enhanced the project in order to check the calculations. I've added the possibility to initialize the neural network with weights and bias:
image

 In addition, the weights and bias are shown in the diagram:
image

I figured it might be useful for others as well to have these features and I would like to commit them into your repository. I hope you feel so too! For changes and/or suggestions free to contact me.

### CODE REVIEW

**Comment by PetrKubes97** [comment]
@Wissiak Sorry, I've merged it prematurely, I like the idea, but a few things make me revert when I was testing it.

- Biases are just weights to neurons, so I'd put the weights to them to the weights list.
- The sliders are 0..1, so it's better to keep everything normalized and not use negative or numbers > 1
- The default setting should converge to a 0 cost.

**Comment by PetrKubes97** [comment]
Feel free to open another PR with the changes. I've also cleaned up the repo, so that the diff isn't insane.
